packages/server/lib/crons/utils/batchDelete.ts

```diff
@@ -0,0 +1,31 @@
+import { setTimeout } from 'node:timers/promises';
```

Comment:
I'll rename the `crons/utils` subfolder to `crons/delete`. utils is definitely overused